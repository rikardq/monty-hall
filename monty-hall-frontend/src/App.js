import React, { Component } from 'react';
import monty from './monty.jpg';
import './App.css';
import BackendHealth from './BackendHealth/BackendHealth';
import MontyHallContainer from "./components/MontyHallContainer";

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <BackendHealth/>
          <img src={monty} className="App-monty" alt="logo" />
          <h1 className="App-title">Welcome to the Monty Hall Simulator</h1>
        </header>
          <MontyHallContainer/>
      </div>
    );
  }
}

export default App;
