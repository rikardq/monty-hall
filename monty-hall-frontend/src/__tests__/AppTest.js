import React from 'react';
import App from '../App.js';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

it('renders without crashing', () => {
    const component = shallow(<App>Facebook</App>);
    expect(component).toMatchSnapshot();
});
