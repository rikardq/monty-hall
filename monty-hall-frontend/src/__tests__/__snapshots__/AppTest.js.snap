// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <BackendHealth />
    <img
      alt="logo"
      className="App-monty"
      src="monty.jpg"
    />
    <h1
      className="App-title"
    >
      Welcome to the Monty Hall Simulator
    </h1>
  </header>
  <MontyHallContainer />
</div>
`;
